src/cookie-io.js

```diff
@@ -170,7 +170,6 @@ class CharacterData {
     setQuestCompleted(scene) {
         if (this.checkScene(scene)) {
             this.characterData.levels[scene].questCompleted = true;
-
```

Comment:
I personally like leaving an empty line above comments so they function as headers, not the end of the world though

Reply:
Reverted!